### ISSUE TITLE
Allow users to select different DockPanelSuite Themes

### ISSUE THREAD

**Issue by kmscode** [mRemoteNG/mRemoteNG#266]
[DockPanelSuite.ThemeVS2012](https://www.nuget.org/packages/DockPanelSuite.ThemeVS2012/) allows the following themes:
* VS2012LightTheme
* VS2012BlueTheme
* VS2012DarkTheme

Allow these to be user selectable.

Currently defaulting to VS2015LightTheme.

In frmMain.cs:
```
private frmMain()
{
	_showFullPathInTitle = Settings.Default.ShowCompleteConsPathInTitle;
	InitializeComponent();
	Fullscreen = new MiscTools.Fullscreen(this);
	pnlDock.Theme = new VS2012LightTheme();
}
```

**Comment by kmscode** [comment]
depends on #269

**Comment by jotatsu** [comment]
Theme selection was added on pull 671. I think this one can be closed.

**Comment by kmscode** [comment]
Fixed via #671 - included in 1.76. Closing.
